connectors/egeria/src/main/java/com/ibm/egeria/EgeriaClient.java

```diff
@@ -55,7 +55,7 @@
     private String assetGuid;
     // at this point we make it a constant, this will change and should be passed as
     // credentials for Egeria
-    private final String userid = "calliequartile";
+    private final String userid = "garygeeke";
```

Comment:
I think the username should not be hard-coded but rather be passed through env variable (a secret attached to the deployment)

Reply:
fixed. Now we have an env variable "EGERIA_DEFAULT_USERNAME" which is set in in egr_connector.yaml. Thanks.